test/puppetlabs/trapperkeeper/authorization/testutils.clj

```diff
@@ -0,0 +1,36 @@
+(ns puppetlabs.trapperkeeper.authorization.testutils
+  (:require [puppetlabs.ssl-utils.core :as ssl])
+  (:import (org.joda.time DateTime Period)))
+
+(defn request
+  "Builds a ring request"
+  [path method certificate ip]
+  {:uri path :request-method method :remote-addr ip :ssl-client-cert certificate})
+
+;; Exctracted from ssl-utils test
+(defn- generate-not-before-date []
+  (-> (DateTime/now)
+      (.minus (Period/days 1))
+      (.toDate)))
+
+(defn- generate-not-after-date []
+  (-> (DateTime/now)
+      (.plus (Period/years 5))
+      (.toDate)))
+
+(defn create-certificate
```

Comment:
I have those exact changes done locally.  Should come in with my query-params PR as a cleanup MAINT commit.